Moco/Moco/MocoGoal/MocoSumSquaredStateGoal.h

```diff
@@ -25,9 +25,21 @@ namespace OpenSim {
 
 /// Minimize the sum of squared states, integrated over the phase. For example, 
 /// this can be used to minimize muscle activations, as is done in MocoInverse.
-/// This class also allows you to select which states to minimize by using
-/// a regex pattern with `setPattern()` and to provide weights for each
-/// state through `setWeightSet()` and `setWeightForState()`.
+///
+/// This goal is computed as follows:
+///
+/// \f[
+/// \int_{t_i}^{t_f} \sum_{s \in S} w_s y_s(t)^2 ~dt
+/// \f]
+/// We use the following notation:
```

Comment:
I think it's clear as is, or for completeness could add what `t_i` and `t_f` represent

Reply:
Addressed.